src/ur.ts

```diff
@@ -16,6 +16,10 @@ export default class UR {
     return new UR(cborEncode(buf));
   }
 
+  public static from(value: any) {
+    return UR.fromBuffer(Buffer.from(value));
```

Comment:
This will not work for strings not encoded in `utf-8`, as `Buffer.from` needs the encoding as a second param. 
Adding it as an optional second param will fix that.
```suggestion
  public static from(value: any, encoding?: string) {
    return UR.fromBuffer(Buffer.from(value, encoding));
```